src/components/table-header.tsx

```diff
@@ -105,26 +185,53 @@ const TableHeader = ({ config }: TableHeaderProps): JSX.Element => {
         </span>
       </div>
       <div className='col-xs-12 col-sm-6'>
-        <div className='pull-right'>
-          {/* <button ng-if='showAddRecord()'
-                className='chaise-table-header-create-link chaise-btn'
-                ng-className='vm.config.displayMode === recordsetDisplayModes.fullscreen ? 'chaise-btn-primary': 'chaise-btn-secondary''
-                ng-click='addRecord()' tooltip-placement='bottom-right' uib-tooltip='Create new {{vm.displayname ? vm.displayname.value : vm.reference.displayname.value}}'>
-                <span className='chaise-btn-icon glyphicon glyphicon-plus'></span>
-                <span>{{config.displayMode === recordsetDisplayModes.fullscreen ? 'Create' : 'Create new'}}</span>
-            </button>
-            <button ng-if='vm.config.displayMode === recordsetDisplayModes.fullscreen && canUpdate()'
-                className='chaise-table-header-edit-link chaise-btn chaise-btn-primary'
-                ng-click='editRecord()' ng-disabled='{{vm.pageLimit > vm.RECORDEDIT_MAX_ROWS}}' tooltip-placement='bottom-right'
-                uib-tooltip='{{ ((vm.pageLimit > vm.RECORDEDIT_MAX_ROWS) ? 'Editing disabled when items per page > ' + vm.RECORDEDIT_MAX_ROWS : 'Edit this page of records.' ) }}'>
-                <span className='chaise-btn-icon glyphicon glyphicon-pencil'></span>
+        <div
+          className='pull-right'
+        >
+          {shouldShowCreateButton() && (
+            <OverlayTrigger
+              placement='bottom-end'
+              overlay={
+                <Tooltip>
+                  Create new{' '}
+                  {reference.displayname && reference.displayname.value}
```

Comment:
Use the `DisplayValue` component here and pass the `displayname` as the "value" prop.